components/MetadataCreation/TextInput.tsx

```
@@ -2,19 +2,36 @@ import { useZoraCreateProvider } from "@/providers/ZoraCreateProvider";
+        disabled={Boolean(fileUploading || creating)}
+        onChange={(e: ChangeEvent<HTMLTextAreaElement>) => {
+          setName(
+            `${e.target.value.slice(0, 10)}${e.target.value.length > 10 ? "..." : ""}`,
+          );
+          setWritingText(e.target.value);
+        }}
+      />
+      <div
+        className="p-4 absolute z-[1] left-0 top-0 bg-white"
+        ref={writingRef}
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)